docs/topic/authenticator-configuration.rst

```diff
@@ -24,6 +24,12 @@ example.
 Configuring the authenticator
 =============================
 
+Some authenticators have unique configuration options. This section covers a
```

Comment:
This document talks about configuring generic authenticators without making specific comments about anything - LDAPAuthenticator is only used as an example. If you found that confusing, we should restructure this quite a bit - however, I think the characterization as 'a few common ones' is not correct.

For auth, I think we will have three types of documentation:

1. Tutorials with step by step for an authenticator, including screenshots (Such as #54). We don't have any yet.
2. How-to guides for specific authenticators, such as what we have for dummyauthenticator
3. This document, which is the 'fall through' when we don't have tutorials or howtos for an authenticator.

I think LDAPAuthenticator is common enough to be a HOWTO. Maybe we should use something else as an example here?

We should also probably put this comment in contributing/doc.rst.

Reply:
see my comments above re: restructuring. You are correct that the main reason I spun it out into its own section was because I found it confusing that LDAP was getting a lot of extra attention. I think if we just made it more clear that LDAP was just for example, it could be better.

It sounds like you have a clearer view of how the pieces fit together here, do you wanna make a push to this PR with how you'd like it to be structured?